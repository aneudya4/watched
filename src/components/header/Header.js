import React from 'react';
import PropTypes from 'prop-types';

const Header = ({ children }) => <header>{children}</header>;
export default Header;

Header.propTypes = {
  children: PropTypes.node.isRequired,
};
