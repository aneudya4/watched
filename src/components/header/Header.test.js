/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('Test on the <Header/> Component', () => {
  it('renders App without crashing', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
