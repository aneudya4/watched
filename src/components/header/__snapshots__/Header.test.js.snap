// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on the <Header/> Component renders App without crashing 1`] = `<header />`;
